src/helper/HelperSignalHandler.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2021 Aleix Pol i Gonzalez <aleixpol@kde.org>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ */
+
+#ifndef HELPERSIGNALHANDLER_H
+#define HELPERSIGNALHANDLER_H
+
+#include <QCoreApplication>
+#include <signal.h>
+
+// ensure we exit gracefully and close the session if sigtermed (i.e restarting sddm)
+static void sigtermHandler(int signalNumber)
+{
+    Q_UNUSED(signalNumber)
```

Comment:
Both commit message and comment says "SIGTERM" yet we ignore the signal all together - SIGTERM or otherwise.
Why is that - can we tweak the comment to mention the reason?